src/eos-dropbox-app

```diff
@@ -121,8 +121,13 @@ class DropboxLauncher():
         self._open_dropbox_directory()
 
     def _name_lost(self, *args):
-        logging.info("Another instance of dropbox is already running")
-        self._already_running = True
+        if not get_dropbox_directory():
+            logging.info("Another instance of dropbox is already running but no "
+                         "Dropbox folder was found; launching the daemon again")
+            self._launch_dropbox_daemon()
```

Comment:
Does this actually work? Since recent versions of dropbox I've noticed that trying to run the daemon again while already running would show an error to the user indicating that "another daemon was already running", but perhaps that behaviour changed once again now?